client/src/pages/private/SiteTable.js

```diff
@@ -24,15 +25,47 @@ import { FeatureFlaggedComponent, flagKeys } from '../../services';
 import { fetchRegionSiteRows, fetchSiteRows } from '../../services/site';
 import { useTableStyles } from '../../components/tables/DataTable';
 
+const allocationStyle = {
+  allocation: {
+    fontWeight: 700,
+  },
+  dates: {
+    color: '#272833',
+    fontWeight: 400,
+  },
+};
+
+const renderAllocations = (row) => {
```

Comment:
Good call - I am also of two minds here. When I know the component can be reused, I will always make a new component, but wasn't sure if it was necessary for this one since it seemed very specific to this table. However, the components are all pretty small, so I'll keep it that way!

Reply:
Sounds good.